tests/nnstreamer_converter/unittest_converter.cc

```diff
@@ -184,6 +185,157 @@ TEST (tensorConverterCustom, invalidParam3_n)
   EXPECT_NE (0, nnstreamer_converter_custom_unregister ("tconv"));
 }
 
+/** @brief tensor converter plugin's query caps callback */
+static GstCaps *
+conv_query_caps (const GstTensorsConfig *config)
+{
+  return NULL;
+}
+
+/** @brief tensor converter plugin's get out caps callback */
+static gboolean
+conv_get_out_config (const GstCaps *in_cap, GstTensorsConfig *config)
+{
+  return TRUE;
+}
+
+/** @brief tensor converter plugin's convert callback
+ */
+static GstBuffer *
+conv_convert (GstBuffer *in_buf, gsize *frame_size, guint *frames_in, GstTensorsConfig *config)
+{
+  return NULL;
+}
+
+/**
+ * @brief Test for plugin registration
+ */
+TEST (tensorConverter, subpluginNoraml)
+{
+  const gchar *name = "name";
+  NNStreamerExternalConverter *sub = g_new0 (NNStreamerExternalConverter, 1);
+  ASSERT_TRUE (sub != NULL);
+
+  sub->name = (char *) name;
+  sub->query_caps = conv_query_caps;
+  sub->get_out_config = conv_get_out_config;
+  sub->convert = conv_convert;
+
+  EXPECT_TRUE (registerExternalConverter (sub));
+
+  g_free (sub);
+}
+
+/**
+ * @brief Test for plugin registration with invalid param
+ */
+TEST (tensorConverter, subpluginInvalidParam0_n)
+{
+  EXPECT_FALSE (registerExternalConverter (NULL));
+}
+
+/**
+ * @brief Test for plugin registration with invalid param
+ */
+TEST (tensorConverter, subpluginInvalidParam1_n)
+{
+  NNStreamerExternalConverter *sub = g_new0 (NNStreamerExternalConverter, 1);
+  ASSERT_TRUE (sub != NULL);
+
+  sub->name = NULL;
+  sub->query_caps = conv_query_caps;
+  sub->get_out_config = conv_get_out_config;
+  sub->convert = conv_convert;

```

Comment:
It seems that codes are somewhat redundant. How about declaring common function returning `sub`?
For example,
```
NNStreamerExternalConverter *sub = get_default_converter ();
sub->name = NULL;
```
I think this approach is easy to understand what makes a failure.

Reply:
Nice! Updated.